examples/PalmerPenguins.jl

```diff
@@ -0,0 +1,65 @@
+using PalmerPenguins, Plots
+const TABLE = PalmerPenguins.load()
+
+using DataFrames, Random
+df = dropmissing(DataFrame(TABLE))
```

Comment:
It doesn't seem very useful to create a constant that is used only once?
```suggestion
using DataFrames, Random
df = dropmissing(DataFrame(PalmerPenguins.load()))
```